src/vm/memory/relocatable.zig

```diff
@@ -564,3 +900,11 @@ test "fromU64: should create a MaybeRelocatable from a u64" {
         fromU64(@intCast(45)),
     );
 }
+
+fn maybeRelocatableRelocatable(segment_index: i64, offset: u64) MaybeRelocatable {
+    return MaybeRelocatable{ .relocatable = Relocatable.new(segment_index, offset) };
+}
+
+fn maybeRelocatableFelt(num: u256) MaybeRelocatable {
+    return MaybeRelocatable{ .felt = Felt252.fromInteger(num) };
+}
```

Comment:
make this helper function because declaring variables was getting very hard, but this name it a bit confusing, suggestions welcome.

Reply:
@0xicosahedron I think that:
- `maybeRelocatableFelt` already exists with 
https://github.com/keep-starknet-strange/cairo-zig/blob/35b888f6bd2496b0c80572f77bd3e2c3fdef9097/src/vm/memory/relocatable.zig#L273-L275
- For `maybeRelocatableRelocatable` you can simplify it by making use of:
https://github.com/keep-starknet-strange/cairo-zig/blob/35b888f6bd2496b0c80572f77bd3e2c3fdef9097/src/vm/memory/relocatable.zig#L255-L257

I mean something like:
```zig
fromSegment(segment_index: i64, offset: u64) MaybeRelocatable {
     return newFromRelocatable(Relocatable.new(segment_index, offset));
}
```

Reply:
Pending this one otherwise looks good to me

Reply:
got it, removed felt variant and hoisted relocatable variant to be near `fromU256` with documentation